weave

```diff
@@ -115,10 +115,35 @@ docker_run_options() {
     echo --privileged --net=host $(docker_sock_options)
 }
 
+docker_exec_cni_options() {
```

Comment:
I avoided mentioning `cni` since `exec_remote()` shouldn't need to know about that.